consumers.md

```diff
@@ -17,6 +17,7 @@ Languages:
 [JavaScript](#javascript),
 [Perl](#perl),
 [Python](#python),
+[Go](#go),
```

Comment:
Like the TAP Producers page, we sort each language alphabetically for easier logical access as the page grows. Could you please move this table of contents entry and the section below to be alphabetically before Java?

Reply:
Not a problem. I'll take another look tomorrow. Thanks!